mex/extractors/pipeline/checks/main.py

```diff
@@ -134,3 +134,42 @@ def check_x_items_more_passed(
     return actual_count <= (historic_count if historic_count > 0 else actual_count) + (
         rule["value"] or 0
     )
+
+
+def check_x_items_less_passed(
+    context: AssetCheckExecutionContext,
+    asset_key: AssetKey,
+    extractor: str,
+    entity_type: str,
+    asset_data: int,
+) -> bool:
+    """Checks whether latest extracted items number is lower than historical count.
+
+    Returns bool to AssetCheck.
+    """
+    yaml_exists = check_yaml_path(extractor, entity_type)
+
+    if not yaml_exists:
+        return True
+
+    rule = get_rule("x_items_less_than", extractor, entity_type)
+    time_delta = parse_time_frame(rule["time_frame"])
+    current_time = datetime.now(UTC)
+    time_frame = current_time - time_delta
+
+    events = context.instance.get_event_records(
+        EventRecordsFilter(
+            asset_key=asset_key,
+            event_type=DagsterEventType.ASSET_MATERIALIZATION,
+        )
+    )
+    if events is None:
+        return True
+
+    actual_count = asset_data
+    historical_events = get_historical_events(events)
+    historic_count = get_historic_count(historical_events, time_frame)
+
+    return actual_count >= (historic_count if historic_count > 0 else actual_count) - (
+        rule["value"] or 0
+    )
```

Comment:
Could you check for the presence of the `rule["value"]` in a separate line?

```suggestion
    if historic_count == 0:
        return True
    return actual_count >= historic_count - rule["value"]
```